.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+tasks:
+  - name: Nats
+    init: |
+      docker-compose pull
```

Comment:
@anbraten Does this have any effect? See https://www.gitpod.io/docs/prebuilds#workspace-directory-only

Reply:
So it has to be a `before` step?

Reply:
You only need `before` if you have something else in `init`. We can just run it `command` i.e. remove it and let `up -d` take care of it.

Reply:
change it to just use a `command`